run-until-success-with-backoff

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -euo pipefail
+
+set -x
+
+i=0
+max_retries=20
```

Comment:
With max_retries 20, we're going from 5 seconds to about 200s. That's x 40.
max_retries 10 brings us to about 50s. x 10 seems plenty.

Reply:
Good idea. Done.